src/porepy/params/parameter_dictionaries.py

```diff
@@ -15,106 +11,107 @@
 def flow_dictionary(g, in_data=None):
     """ Dictionary with parameters for standard flow problems.
 
-    All parameters listed below which are not specified in in_data are assigned unitary
+    All parameters listed below which are not specified in in_data are assigned default
     values. Additional parameters may be passed in in_data.
-    Parameters:
+
+    Args:
         g: Grid.
         in_data: Dictionary containing any custom (non-default) parameter values.
 
     Returns:
         Dictionary with the "mathematical" parameters required by various flow
             discretization objects specified.
     """
-    # Ensure that parameters not handled below are copied or initialize empty dictionary
-    if not in_data:
-        in_data = {}
-    d = in_data.copy()
     # Ensure that the standard flow parameters are present in d. Values from in_data
     # have priority over the default values.
-    d["aperture"] = in_data.get("aperture", np.ones(g.num_cells))
-    d["porosity"] = in_data.get("porosity", np.ones(g.num_cells))
-    d["fluid_compressibility"] = in_data.get(
-        "fluid_compressibility", np.ones(g.num_cells)
-    )
-    d["mass_weight"] = in_data.get("mass_weight", d["porosity"])
-    d["source"] = in_data.get("source", np.zeros(g.num_cells))
-    d["time_step"] = in_data.get("time_step", 1)
-    # Set the second order tensor from the permeability.
-    d["second_order_tensor"] = in_data.get(
-        "permeability", pp.SecondOrderTensor(g.dim, np.ones(g.num_cells))
-    )
-    d["bc"] = in_data.get("bc", pp.BoundaryCondition(g))
-    d["bc_values"] = in_data.get("bc_values", np.zeros(g.num_faces))
+    d = {
+        "aperture": np.ones(g.num_cells),
+        "porosity": np.ones(g.num_cells),
+        "source": np.zeros(g.num_cells),
+        "mass_weight": np.ones(g.num_cells),
+        "second_order_tensor": pp.SecondOrderTensor(g.dim, np.ones(g.num_cells)),
+        "bc": pp.BoundaryCondition(g),
+        "bc_values": np.zeros(g.num_faces),
+        "fluid_compressibility": np.ones(g.num_cells),
+        "time_step": 1,
+    }
```

Comment:
mass_weight and fluid_compressibility should maybe be merged into one parameter? Anyway, this is not the pullrequest for doing this